manifests/repo/redhat.pp

```diff
@@ -23,9 +24,9 @@
     priority   => 1,
   }
 
-  yumrepo { 'remi-php56':
-    descr      => 'Remi\'s PHP 5.6 RPM repository for Enterprise Linux $releasever - $basearch',
-    mirrorlist => "https://rpms.remirepo.net/enterprise/${releasever}/php56/mirror",
+  yumrepo { "remi-php${no_dot_version}":
```

Comment:
There might be leftovers when we change `$version`, is it safe or can it be an issue?

Reply:
It is hard to say.

Reply:
I'm not really sure how to go about cleanup if a version is changed.   I would just assume leave it up to the end-user to do an ensure absent in their control repo.   Open to suggestions though.